routes/userManage.js

```diff
@@ -4,7 +4,17 @@ const express = require('express');
 const router = express.Router();
 const {manage} = require ('../controllers/userManagementController');
 
-//Route to create new Profile or update existing profile
-router.post('/manage', manage);
-
-module.exports = router;
+export default async function handler(req, res) {
```

Comment:
You could have made use of the express function router.post() instead of this, but this is fin for now, we can get back to this later